api/forbidden_shop.php

```diff
@@ -45,6 +48,17 @@
             $message = $ForbiddenShopManager->buyForbiddenJutsu($jutsu_id);
             $player->getInventory();
 
+            $response->data = [
+                'message' => html_entity_decode($message, ENT_QUOTES),
+                'playerInventory' => UserApiPresenter::playerInventoryResponse($player)
+            ];
+            break;
+        case "exchangeFavor":
+            $item_id = $system->db->clean($_POST['item_id']);
```

Comment:
If this is a numeric ID, you could do `(int)$_POST['item_id']` instead

Reply:
Fixed in commit.